metricshub-web/react/src/store/thunks/configThunks.js

```diff
@@ -9,7 +15,13 @@ export const fetchConfigList = createAsyncThunk(
 	"config/fetchList",
 	async (_, { rejectWithValue }) => {
 		try {
-			return await configApi.list();
+			// Fetch config files and backups separately and merge
+			const [configs, backups] = await Promise.all([
+				configApi.list(),
+				configApi.listBackups().catch(() => []),
```

Comment:
Why do we catch errors after listBackups()? It’s likely due to a backend issue where the backup folder cannot be found.